validate.go

```diff
@@ -53,23 +56,23 @@ var bundleValidateCommand = cli.Command{
 			logrus.Fatal(err)
 		}
 
-		sf, err := os.Open(path.Join(inputPath, "config.json"))
+		content, err := ioutil.ReadFile(path.Join(inputPath, "config.json"))
 		if err != nil {
 			logrus.Fatal(err)
 		}
-
-		defer sf.Close()
-
+		if !utf8.Valid(content) {
+			logrus.Fatalf("'Config.json' is not encoded in UTF-8")
```

Comment:
Actually, maybe store the `path.Join` in a variable, and pass that variable to both `ReadFile` and this error message.